packages/odyssey-react/src/components/Table/TableContainer.module.scss

```diff
@@ -0,0 +1,31 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+.container {
```

Comment:
Same as above

> Why no `.root` here?

Reply:
@arnoldsandoval I just missed changing the class names when I split apart the style modules - good catch. If you don't mind I can fix that in a follow up PR after this bigger one goes thru to avoid merge conflicts!

Reply:
Not a problem!